qt/python/mantidqt/widgets/sliceviewer/peaksviewer/representation/painter.py

```diff
@@ -14,6 +14,7 @@
 
 # constants
 ZOOM_PAD_FRAC = 0.2
+MIN_PAD = 0.05
```

Comment:
Could we comment what these constants do whilst were here?

Reply:
They are constant 😉 
I'll add better comments there :)